Resources/config/hydra.xml

```diff
@@ -22,10 +28,12 @@
             <tag name="kernel.event_listener" event="kernel.response" method="onKernelResponse" />
         </service>
 
-        <service id="api.hydra.listener.request_exception" class="Dunglas\ApiBundle\Hydra\EventListener\RequestExceptionListener">
-            <argument type="service" id="api.hydra.normalizer.error" />
+        <service id="api.hydra.listener.exception" class="Symfony\Component\HttpKernel\EventListener\ExceptionListener">
+            <argument>api.hydra.controller.exception:showAction</argument>
```

Comment:
The original listener define this as a `%twig.exception_listener.controller%` parameter. Why recreating the service ?